analyser/main.py

```
@@ -250,16 +344,19 @@ def compute_temporal_embedding(wav_path, model):
                         "id": sampleId,
                         "url": sampleUrl,
                         "path": temp_filename,
+                        "filename": filename,
                         "missing": False
                     })
                 except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)